geoips/filenames/base_paths.py

```diff
@@ -130,7 +121,7 @@
 if getenv('GEOIPS_RCFILE'):
     PATHS['GEOIPS_RCFILE'] = getenv('GEOIPS_RCFILE')
 
-PATHS['TC_TEMPLATE'] = pathjoin(PATHS['GEOIPS'], 'geoips', 'yaml_configs', 'sectors_dynamic', 'tc_web_template.yaml')
+PATHS['TC_TEMPLATE'] = pathjoin(PATHS['BASE_PATH'], 'yaml_configs', 'sectors_dynamic', 'tc_web_template.yaml')
```

Comment:
@jsolbrig Is this how these will be handled when we have added full support for entry points package_data, or is this just a first step (ie, getting rid of $GEOIPS)? (just curious)

Reply:
I think so, yes. We need to look more closely at this.